src/features/Movies/componentPreviousVersion.js

```diff
@@ -0,0 +1,175 @@
+/* eslint-disable max-len */
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+// import components
+import MovieCategories from '../MovieCategories';
+import MovieItem from '../MovieItem';
+import MovieFilters from '../MovieFilters';
+import AddMovie from '../../shared/AddMovie';
+import {filterArray} from '../../assets/functions/util';
+// import mockData 
+import { mockData } from '../../assets/data/MockData';
+import { v4 as uuidv4 } from 'uuid';
+import './style.scss';
+
+export default class Movies extends Component {
+    // define our state
+    constructor(props){
```

Comment:
missed this component? I expected we transforming all components to Functional

Reply:
I kept this one as a reference for me, it is not imported anywhere in the code